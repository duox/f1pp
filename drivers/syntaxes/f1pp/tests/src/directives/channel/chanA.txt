#5:A
#6:B
#1 "chanA.txt":
#5:A
#6:B
#8 "test.c":
#9:C