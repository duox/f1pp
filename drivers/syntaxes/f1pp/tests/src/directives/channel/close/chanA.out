#5:A
#7:B